pkg/environment/environment.go

```diff
@@ -179,6 +182,29 @@ func GetHelmWaitTimeout() (time.Duration, error) {
 	return time.Duration(timeout) * time.Second, nil
 }
 
+// GetDiscoveryBurst returns the K8s discovery burst value if it is set, otherwise it returns -1
+func GetDiscoveryBurst() (int, error) {
+	burstStr := os.Getenv(DiscoveryBurst)
+	if burstStr == "" {
+		return -1, nil
+	}
+	return strconv.Atoi(burstStr)
```

Comment:
In case of an error, 0 is returned as a value.

Reply:
thanks, fixed